### DISCUSSION TITLE
迷你世界小故事，老玩家必看

### DISCUSSION THREAD

**Discussion by Modog1145** [share121/inter-knot#151] [其他]
![8445517232041022](https://github.com/user-attachments/assets/73f821d5-70e8-4e39-8cc8-a4450421f77d)

[https://b23.tv/d3HZaES](url)
咋办呢咋办呢😭
暑假就要结束了😱
我暑假作业还没写完呢😇
老师留那么多作业😡
我咋写呀😵‍💫
先玩会迷你世界吧😝
这是我抓的拆家军的成员😆
给他们都关笼子里头🤓
让他们老炸我的房子🤪
哭什么哭啊😅
让你哭了吗啊🤬
给我老实点😋
你们都给我老实点🤤
知道吗😓
啊这是小美😍
每天都被拆家军欺负😤
今天我把你保护起来😃
这样你就不会受到伤害了🥳
还有我养的大犀牛😯
我养了两年半的大犀牛😗
有红色的👹
有蓝色的🥶
真是太可爱了😘
还有我种的小花😁
真是太好了😎
就在这大别墅里一待🤗
还学啥习🥴
写啥作业呀🤭
我就不学习😛
我就玩迷你世界🙄
哎我去💀
又是你们这些可恶的拆家军🤯
小美😰
别打我的宠物鸡🤮
小美🤒
我的房子🤐
你们干啥呀😵‍💫
你们这些可恶的拆家军😖
又给我的房子拆了☹️
我写作业还不行吗🤕
别拆了😟
我现在就写作业😔
别拆了☠️
我的房子😒
别炸😠
我的大别墅都被你们给炸坏了🤧
你们干啥呀😧
炸吧😂
我也炸😜
我也要加入拆家军😊
加入拆家军就不用写作业了😮‍💨
我也炸🥵
迷你世界是全天下最好玩的游戏🫡
嘎嘎嘎嘎🤠
诶🤔

**Comment by dveam** [comment]
老丸家看哭了😭
